runtime/onert/backend/train/TensorManager.cc

```diff
@@ -43,6 +43,14 @@ void allocateMemory(backend::train::MemoryManager *mgr,
   }
 }
 
+size_t alignedSize(const size_t size, const uint64_t align)
+{
+  assert(size);
+  const size_t aligned = ((size - 1) | (align - 1)) + 1;
+  assert(aligned % 16 == 0);
+  return aligned;
+}
+
```

Comment:
Though it seems size = 0 is not allowed for valid input, we may use the traditional form of:
```cc
#define align_up(num, align) \
    (((num) + ((align) - 1)) & ~((align) - 1))
```
It returns zero when size = 0.

Reply:
Macros are not type safe. So I think it's better to use an inline function although inline functions are not always guaranteed to be inlined.

Reply:
Ah, I am not saying to use `macro`. Just find the implementation body.